install.sh

```diff
@@ -1,4 +1,10 @@
 set -euo pipefail
+VAR_DIR='/var/lib/tensorleap/standalone'
+SCRIPT_LOG=$VAR_DIR/install_log.log
+
+function log(){
+  echo "[$(date)] [INFO]  $1" >> $SCRIPT_LOG
```

Comment:
Looks like extra space
```suggestion
  echo "[$(date)] [INFO] $1" >> $SCRIPT_LOG
```